src/helpers/Utility.ts

```diff
@@ -19,4 +19,9 @@
  */
 
 class Utility {
+ 
+    private pad(num: number): string {
```

Comment:
this method needs to be accessible outside this class.

Reply:
also, we need to have this as a static method so that we can access it without creating an instance of Utility